composer.json

```diff
@@ -19,6 +19,9 @@
         "drupal/migrate_tools": "^5.0",
         "drupal/ultimate_cron": "^2.0@alpha"
     },
+    "require-dev": {
+        "drupal/migrate_source_csv": "^3.4"
```

Comment:
Should this be moved to the regular `require` object instead of `require-dev`?

Reply:
It's not a hard requirement. the routes and everything allow it to be an optional dependency.